NHSD.BuyingCatalogue.Ordering.Services/NHSD.BuyingCatalogue.Ordering.Services.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0"?>
+<doc>
+    <assembly>
+        <name>NHSD.BuyingCatalogue.Ordering.Services</name>
+    </assembly>
+    <members>
+    </members>
+</doc>
```

Comment:
We shouldn't have the XML docs files checked-in. Can you delete this file?

The changes requested to the project files below should prevent this file from being generated.

Reply:
Removed XML file

Reply:
Deleted folder `NHSD.BuyingCatalogue.Ordering.Services` from solution root